docs/en_US/reference/experiment_config.rst

```diff
@@ -360,9 +372,9 @@ type: list of `RemoteMachineConfig`_
 reuse mode
 ----------
 
-Enable reuse mode [TODO]
+Enable reuse mode [TODO: doc for reuse mode]
```

Comment:
maybe you can use anchor, like 
```
`reuse mode <../Tutorial/ExperimentConfig.rst#reuse>`__
```

Reply:
The problem is nobody will use the feature with "doc" like this...